src/github/github.controller.ts

```diff
@@ -0,0 +1,225 @@
+import {
+  Controller,
+  Query,
+  Get,
+  HttpStatus,
+  Post,
+  UseGuards,
+  HttpException,
+  Req,
+  Put,
+  Param,
+  Body,
+  Delete,
+} from '@nestjs/common';
+import { GithubService } from './github.service';
+import { ApiBearerAuth, ApiTags } from '@nestjs/swagger';
+import { Env } from 'src/env';
+import { JwtAuthGuard } from 'src/auth/jwt-auth.guard';
+import { UpdatePracticesDto } from './dto/update.practices.dto';
+import { TRequest } from 'src/common/types/user.types';
+import { GetApiQueryParamsSwaggerWithOrganizationName } from 'src/decorator/decorator';
+
+@Controller('github')
+@ApiTags('Github')
```

Comment:
PullProofAI

Consider adding more specific error handling in the `listRepos` method to differentiate between different types of errors for better debugging and user feedback.

```diff
+       // Add specific error handling based on error types
```


<!-- This is an auto-generated comment by PullProofAI -->